### ISSUE TITLE
C++ I/O exception on Fedora 23

### ISSUE THREAD

**Issue by standage** [dib-lab/khmer#1368]
I get the following issue when running `make test` with the current master on Fedora 23.

```
tests.test_countgraph.test_load_truncated ... terminate called after throwing an instance of 'std::ios_base::failure'
  what():  basic_ios::clear
```

**Comment by luizirber** [comment]
I'm seeing similar errors with Arch Linux, seems to be a change on GCC 5
stdlib

**Comment by standage** [comment]
Hmm, I thought this would have been fixed by #1407, but now on Fedora 23 I'm getting:

```
tests/test_countgraph.py::test_load_truncated Makefile:295: recipe for target 'test' failed
make: *** [test] Aborted (core dumped)
```
